include/LightGBM/feature_group.h

```diff
@@ -181,6 +185,8 @@ class FeatureGroup {
   std::unique_ptr<Bin> bin_data_;
   /*! \brief True if this feature is sparse */
   bool is_sparse_;
+  /*! \brief Threshold for treating a feature as a sparse feature */
+  double sparse_threshold_;
```

Comment:
The reason I put `sparse_threshold` into `FeatureGroup` is that a very related parameter, `is_enable_sparse`, is also a member per `FeatureGroup` rather than per `Dataset`. When I was adding `sparse_threshold` I wanted to manage the two parameters in the same way. 

I can certainly put sparse_threshold into `Dataset` instead if you request me to do so. But is there a reason why `is_enable_sparse` is not put into `Dataset`? Or maybe its just an imperfection of the code?